README.md

```diff
@@ -10,24 +16,41 @@ Videotools depends on having textmate installed. It will attempt to install this
+**VideoAIP Configuration:** 
 
-This script includes an option to sync package/access copies to a remote or local location for easy backup.
+When you run `videoaip -e` you will see the following configuration screen:
 
-Usage: `videoaip [option] [inputfile1] [inputfile2] ... `, help: `videoaip -h`, crop mode: `videoaip -c`,set up configuration for options `videoaip -e`
+![Video AIP Config](https://github.com/pugetsoundandvision/audiotools/blob/master/supplemental/VideoAIP_Config.png)
+
+* Option One is to enable synchronizing (copying) your preservation package to a second location. Enable this option by entering `Y` for `sync_choice` (between quotation marks) and a path after `destination` (between parenthesis).
+
+* Option Two: VideoAIP will still create a preservation package at the same location as the original file, even if you have enabled synchronization in Option One. If you wish to delete this package once a copy has been synced to the destination specified above, enter `Y` after `remove_aip`.
+
+* Option Three allows you to specify a location for an additional copy of your access files. This enables you to have copies of all access files added to one central folder for more easy management. Enable this by entering `Y` and a path (or ssh path) following `derivative_destination`.
+
+Example:
+
+Original .MKVs are on HardDrive_A. We want the preservation packages synced to HardDrive_B, and copies of the access files saved to HardDrive_C. We also want the preservation packages on HardDrive_A to be deleted once they are synced to HardDrive_B.
+
+![VideoAIP Example](https://github.com/pugetsoundandvision/audiotools/blob/master/supplemental/VideoAIP_Example_GitHub.png)
+
+**Creating Preservation Packages:**
+
+Before you begin:
+
+* Each video file and it's associated logs should be together in one folder. (I.e., FileA.mkv, LogA.log, FrameA.framemd5 should all be in "FileA" folder).
+
+If you only have video files, you can simply run:
 
-**VideoAIP Configuration:**
+`videoaip [drag each .mkv file path here, separated by spaces]`
 
-When you run `videoaip -e` you will see a configuration screen like this:
-![AudioAIP Config](https://github.com/pugetsoundandvision/audiotools/blob/master/supplemental/audioaipconfig.png)
+This will create an archival package for each .mkv that adheres to the bagit standard with an mp4 access file, technical metadata, and checksums.
 
-To set your options, simply type them between the respective quotation marks, save the file and quit.
+If you have logs that are associated with your video files, such as .log or .framemd5 files, you can run:
 
-The options require file paths.  An easy way to find the file path for a file or folder is just to drag it into the terminal window and then copy the results. It should looks something like `/Users/username/Desktop/myfolder`
+`videoaip -l auto [drag each .mkv file path here, separated by spaces]`
```

Comment:
probably should note that it can either be `-l auto` or `-l somelog.txt -l someotherlog.txt...` for the manual option